Documentation/devices.md

```diff
@@ -73,6 +73,7 @@
 - BQ X5 Plus (Longcheer L9360)
 - Leeco s2
 - Motorola Moto G5 (cedric) (quirky - see comment in `lk2nd/device/dts/msm8952/msm8937-motorola-cedric.dts`)
+- HMD Global Nokia 6 (ple)
```

Comment:
```suggestion
- BQ X5 Plus (Longcheer L9360)
- HMD Global Nokia 6 (ple)
- Leeco s2
- Motorola Moto G5 (cedric) (quirky - see comment in `lk2nd/device/dts/msm8952/msm8937-motorola-cedric.dts`)
```

I guess you sorted it with N for Nokia but let's keep the list sorted fully alphabetically :)

Reply:
Yeah, you guessed correctly; just forgot about the "HMD Global" part when sorting it.